duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -36,8 +34,9 @@ public class Http {
   private final String host;
   private final String uri;
   private final String signingAlgorithm = "HmacSHA512";
+  private final String hashingAlgorithm = "SHA-512";
   private Headers.Builder headers;
-  Map<String, String> params = new HashMap<String, String>();
+  private Map<String, String> params = new TreeMap<String, String>();
```

Comment:
Can we make the left side SortedMap<String, String>?  that will indicate we're relying on the sorting.